### PR TITLE
Added new delegation controller for frontend

### PR DESCRIPTION
**Pull request by acn-dgopa**
Added new controller for frontend (backend for frontend), modified so…me test data setup and duplicated tests for bff delegation

## Related Issue(s)
- #{issue number}

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/Altinn.AccessManagement/Controllers/BFF/DelegationsController.cs [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        }
+
+        /// <summary>
+        /// Endpoint for retrieving delegated resources between parties
+        /// </summary>
+        /// <response code="400">Bad Request</response>
+        /// <response code="500">Internal Server Error</response>
+        [HttpGet]
+        [Authorize]
+        [Route("accessmanagement/api/v1/bff/{who}/delegations/maskinportenschema/inbound")]
```
Comment (TheTechArch) [inline src/Altinn.AccessManagement/Controllers/BFF/DelegationsController.cs:56]: Maybe change api to bffapi so the url change is easier, and it is easier to differentiate from the real api

File: src/Altinn.AccessManagement/appsettings.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "BridgeApiEndpoint": "https://at22.altinn.cloud/sblbridge/authorization/api/",
     "JwtCookieName": "AltinnStudioRuntime",
     "OpenIdWellKnownEndpoint": "http://localhost:5101/authentication/api/v1/openid/",
-    "ProfileApiEndpoint": "http://localhost:5101/profile/api/v1/",
+    "ProfileApiEndpoint": "http://platform.at22.altinn.cloud/profile/api/v1/",
```
Comment (TheTechArch) [inline src/Altinn.AccessManagement/appsettings.json:23]: Det bør vel være localtest url her?

**Review by TheTechArch — APPROVED** [review]

File: src/Altinn.AccessManagement/appsettings.json (hunk 2 above)
Comment (acn-dgopa) [reply src/Altinn.AccessManagement/appsettings.json]: stemmer. Endret

File: src/Altinn.AccessManagement/Controllers/BFF/DelegationsController.cs (hunk 1 above)
Comment (acn-dgopa) [reply src/Altinn.AccessManagement/Controllers/BFF/DelegationsController.cs]: ya, updated to bffapi

**Pull request merged**